helm-charts/common/nginx/gateway-values.yaml

```diff
@@ -0,0 +1,243 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Gateway mode example for nginx chart
+# This demonstrates how to use nginx as a central gateway/router
+#
+# IMPORTANT: This is an example only. In practice, you should:
+# 1. Create your own E2E chart that includes this nginx chart as a dependency
+# 2. Define environment variables in your E2E chart's {{ .Release.Name }}-nginx-config ConfigMap
+# 3. Use the nginx template configuration shown below
+#
+# For a complete example, see how chatqna chart implements this pattern.
+
+# Override service type for gateway mode
+service:
+  type: NodePort
+  port: 80
```

Comment:
We don't need to duplicate the same settings as defined in values.yaml.

In gateway-values.yaml, we only needs to put diff here against default values defined in values.yaml